### DISCUSSION TITLE
Why is onload-script no longer supported?

### DISCUSSION THREAD

**Discussion by danielbayley** [npm/rfcs#267] [General] [answered]
> As mentioned [here](https://github.com/npm/cli/blob/latest/CHANGELOG.md#programmatic-usage) in the `CHANGELOG`…
> 
> > The `onload-script` option is no longer supported.
> 
> Why was this removed, or was this kind of functionality replaced with something else? It was useful for example to be able to write `package.json` as `package.yml`, amongst other things, but now it is broken…
> 
> What was the reasoning behind this, and is there some kind of replacement mechanism for things like this?

**Comment by darcyclarke** [comment, marked as answer]
https://github.com/npm/feedback/discussions/71#discussioncomment-116737
